src/screens/BpHistoryScreen.tsx

```diff
@@ -30,14 +30,16 @@ function BpHistoryScreen({navigation, route}: Props) {
     <View style={{flex: 1}}>
       <SafeAreaView
         style={[containerStyles.fill, {backgroundColor: colors.white}]}>
-        <View style={{margin: 24}}>
-          <BodyHeader style={{fontSize: 22, fontWeight: 'bold'}}>
-            BP History
-          </BodyHeader>
-          {bps.map((bp, index) => {
-            return <BpInformation bp={bp} key={index} />
-          })}
-        </View>
+        <ScrollView>
```

Comment:
Can you modify the BP list screen to use a FlatList instead of a ScrollView pls?

https://reactnative.dev/docs/flatlist

It's a more optimised way to build what could potentially be a v long list. Good practice.